"id_", "type_", "affects__0_", "consequence_", "description_", "event_", "isOutputOf__0_", "isOutputOf__1_", "linkTo__0_", "location__type_", "location__coordinates__0__0__0_", "location__coordinates__0__0__1_", "location__coordinates__0__1__0_", "location__coordinates__0__1__1_", "location__coordinates__0__2__0_", "location__coordinates__0__2__1_", "location__coordinates__0__3__0_", "location__coordinates__0__3__1_", "location__coordinates__0__4__0_", "location__coordinates__0__4__1_", "mitigatedBy__0_", "severity_", "sourceRisk__0_", "threat_", "validFrom_", "validTo_", "@context__0_"
"urn:ngsi-ld:Risk:01", "Risk", "urn:ngsi-ld:ServiceGISData:01", "reputation", "Risk01 Corresponds to a failure in the Pumps of the water infraestructure", "interruption", "urn:ngsi-ld:EOGeoDataLayer:01", "urn:ngsi-ld:EOGeoDataLayer:01", "urn:ngsi-ld:Risk:02", "Polygon", "23.6627", "41.88768", "25.85598", "43.38622", "23.4899", "43.78691", "22.35609", "42.28869", "23.6627", "41.88769", "urn:ngsi-ld:Mitigation:01", "MEDIUM", "urn:ngsi-ld:Attacker:01", "physical", "2021-02-18T12:00:00Z", "2021-02-18T12:00:00Z", "https://raw.githubusercontent.com/smart-data-models/dataModel.RiskManagement/master/context.jsonld"